src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -235,19 +222,19 @@ fields for the maps.
 <pipeline_name>/
     sub-<participant_label>/
         dwi/
-            <source_keywords>[_space-<space>][_model-<label>][_desc-<label>]_<map_label>.nii[.gz]
+            <source_keywords>[_space-<space>][_desc-<label>]_model-<label>_<parameter>.nii[.gz]

```

Comment:
Perhaps we can accept this except that we want _model-<model>_parameter-<parameter>_diffusionmodeo.nii[.gz]